app/controllers/katello/concerns/organizations_controller_extensions.rb

```diff
@@ -30,7 +30,7 @@ def create
           if taxonomy_class == Organization
             begin
               @taxonomy = Organization.new(resource_params)
-              sync_task(::Actions::Katello::Organization::Create, @taxonomy)
+              ::Katello::OrganizationCreator.new(@taxonomy).create!
```

Comment:
Should our `create` calls from controllers, um, be in a transaction?

(since we don't have that protection from Dynflow anymore)